woof-code/rootfs-petbuilds/xdg-puppy-labwc/etc/xdg/templates/_root_.config_labwc_menu.xml

```diff
@@ -30,8 +30,8 @@
   <menu id="xdg-menu-Fun" />
   <item label="____________">
   </item>
-  <item label="Exit">
-    <action name="Exit"></action>
+  <item label="Leave">
+    <action name="Execute"><command>logout_gui</command></action>
   </item>
   <item label="Reconfigure">
```

Comment:
I suppose this entry is a bit redundant too, as `logout_gui` refreshes menu and runs `killall -SIGHUP labwc` to restart all configurations.